### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11<br /><br />Workflow run https://github.com/creative-commoners/cwp-pdfexport/actions?query=branch%3Apulls%2F1.3%2Fmodule-standards

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,6 +16,9 @@
         "phpunit/phpunit": "^9.5",
         "squizlabs/php_codesniffer": "^3"
     },
+    "conflict": {
+        "silverstripe/auditor": "<2.4.0"
```
Comment (ChloeSartorelli) [inline composer.json]: Why this conflict? Can you link to a failing CI run or something that caused you to add this?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json]: prefer-lowest build issue
```
 1) CWP\PDFExport\Tests\Extensions\PdfExportExtensionTest::testPdfFilenameWithoutPublicDirectory with data set "no public folder" ('', 'assets/_generated_pdfs/test-p...-1.pdf')
implode(): Passing glue string after array is deprecated. Swap the parameters
/home/runner/work/cwp-pdfexport/cwp-pdfexport/vendor/silverstripe/auditor/code/AuditHook.php:132
```
https://github.com/creative-commoners/cwp-pdfexport/runs/7330242616?check_suite_focus=true#step:12:68

Comment (ChloeSartorelli) [reply composer.json]: That is fixed in 2.2.1 - please update the conflict to `<2.2.1`
https://github.com/silverstripe/silverstripe-auditor/releases/tag/2.2.1

Comment (emteknetnz) [reply composer.json]: Updated

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**